src/helpers/comment.ts

```diff
@@ -9,30 +10,26 @@ type Node = {
   childNodes?: Node[];
 };
 
-const traverse = (result: Record<string, string[]>, node: Node, itemsToExclude: string[]): Record<string, string[]> => {
-  if (itemsToExclude.includes(node.nodeName)) {
-    return result;
-  }
-
+const traverse = (result: Record<string, string[]>, node: Node): Record<string, string[]> => {
   if (!result[node.nodeName]) {
     result[node.nodeName] = [];
   }
 
   result[node.nodeName].push(node.value?.trim() ?? "");
 
-  if (node.childNodes && node.childNodes.length > 0) {
-    node.childNodes.forEach((child) => traverse(result, child, itemsToExclude));
+  if (node.childNodes && node.childNodes.length > 0 && !DefaultConfig.incentives.comment.ignore_children.includes(node.nodeName)) {
```

Comment:
you shouldn't use the default config, but by using `getBotConfig`

Reply:
> you shouldn't use the default config, but by using `getBotConfig`

oh sure

Reply:
@GaryThisSidee